addons/l10n_be_pos_sale/static/src/js/PaymentScreen.js

```diff
@@ -0,0 +1,22 @@
+/** @odoo-module **/
+
+import PaymentScreen from 'point_of_sale.PaymentScreen';
+import Registries from 'point_of_sale.Registries';
+
+export const PoSSaleBePaymentScreen = (PaymentScreen) =>
+    class extends PaymentScreen {
+        async toggleIsToInvoice() {
```

Comment:
```suggestion
        toggleIsToInvoice() {
```

Don't change the "color" of the function, we should keep it synchronous.